packages/api/src/index.ts

```diff
@@ -37,3 +38,14 @@ void (async () => {
   .catch((err) => {
     logger.crit('An error happened during app start', err);
   });
+
+
+  app.get('/login/github', (req, res) => {
+    const url = `https://github.com/login/oauth/authorize?client_id=${CLIENT_ID}&redirect_uri=http://localhost:3000/api/auth/callback/github?&scope=user:email`;
+    res.redirect(url);
+  } );
+
+  app.get('/login/github/callback', (req, res) => {
+    const url = ``;
+    res.redirect(url);
+  } );
```

Comment:
I was going to make the same suggestion, but we need to fix our Actions because they'd be failing for this reason.

As a general rule of thumb, you may want to enable `Format on Save` within VS Code and then make sure the extensions recommended by the repo (in the workspace file) are installed. Aside from that, check the scripts for things like lint/prettier than can be run to auto-fix your formatting problems.